packages/cozy-sharing/src/components/ShareByLink.jsx

```diff
@@ -78,6 +78,28 @@ const ShareByLink = ({ link, document, documentType, onEnable }) => {
     setAlert({ ...alert, open: false })
   }
 
+  const showMobileShareButtons = link && isMobile && navigator.share
+  const showDesktopShareButtons =
```

Comment:
Always confusing to have a var named `Desktop` but can be apply to a `mobile`.

Reply:
Right! And yesterday I read the "Naming" chapter of the Clean Code book...